From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Wed, 26 Aug 2015 13:20:08 +0100
Subject: automake options

Migrate automake options from Makefile.am to configure.ac and add -Wall option,
which has no discernable effect, which is good.
---
 autocrap/Makefile.am  | 2 --
 autocrap/configure.ac | 2 +-
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/autocrap/Makefile.am b/autocrap/Makefile.am
index 556cb42..76b0587 100644
--- a/autocrap/Makefile.am
+++ b/autocrap/Makefile.am
@@ -1,5 +1,3 @@
-AUTOMAKE_OPTIONS=	foreign subdir-objects
-
 bin_PROGRAMS=	scrypt
 dist_man_MANS=$(scrypt_man_MANS)
 
diff --git a/autocrap/configure.ac b/autocrap/configure.ac
index f7f8b8a..e471df6 100644
--- a/autocrap/configure.ac
+++ b/autocrap/configure.ac
@@ -1,6 +1,6 @@
 AC_INIT([scrypt],[1.2.0])
 AC_CONFIG_AUX_DIR([config.aux])
-AM_INIT_AUTOMAKE()
+AM_INIT_AUTOMAKE([foreign subdir-objects -Wall])
 AC_PROG_CC
 AC_PROG_RANLIB
 
